### PR TITLE
Import PyPI Luhn Library

### PR DESCRIPTION
**Pull request by OmegaPointZero**
Imports PyPI Luhn library: https://pypi.org/project/luhn/

### CODE REVIEW

**Review by aslepakurov — APPROVED** [review]
This is so 🔥

File: larky/src/main/resources/vendor/luhn.star [hunk 1]
```diff
@@ -0,0 +1,44 @@
+load("@stdlib//larky", "larky")
+
+def _sum(nums):
```
Comment (mahmoudimus) [inline larky/src/main/resources/vendor/luhn.star:3]: wow, i did not know that we didn't have a sum built-in! do you mind moving this function to the[ builtins module (similar to python?)](https://docs.python.org/3/library/functions.html#sum). that way you can just do:

```python
load("@stdlib//builtins", "builtins")
builtins.sum() # ... https://docs.python.org/3/library/functions.html#sum
```

File: larky/src/main/resources/vendor/luhn.star [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    return ((cardSum + checksum) % 10) == 0
+
+def generate(num):
+    cardSum = int(_luhn_summation(num,-1))
+    modulated = cardSum % 10
+    return 10 - modulated
+
+def append(num):
+    return num + str(generate(num))
+
```
Comment (mahmoudimus) [inline larky/src/main/resources/vendor/luhn.star:38]: can we copy the docstrings as well in the function from here? 

https://github.com/mmcloughlin/luhn/blob/master/luhn.py#L3-L43

File: larky/src/test/resources/vendor_tests/luhn/luhn_test.star [hunk 3]
```diff
@@ -0,0 +1,26 @@
+load("@vendor//luhn", luhn="luhn")
```
Comment (mahmoudimus) [inline larky/src/test/resources/vendor_tests/luhn/luhn_test.star:1]: lets sort this (as per "pep8") and have this after `load("@vendor//asserts", "asserts")`

File: larky/src/main/resources/vendor/luhn.star (hunk 1 above)
Comment (OmegaPointZero) [reply larky/src/main/resources/vendor/luhn.star:3]: Can do!

File: larky/src/main/resources/vendor/luhn.star (hunk 2 above)
Comment (OmegaPointZero) [reply larky/src/main/resources/vendor/luhn.star:47]: Added them:
https://github.com/OmegaPointZero/starlarky/blob/bc58a870acf4683607ee81934ee0ab7538a8ad8e/larky/src/main/resources/vendor/luhn.star#L40-L46

File: larky/src/test/resources/vendor_tests/luhn/luhn_test.star (hunk 3 above)
Comment (OmegaPointZero) [reply larky/src/test/resources/vendor_tests/luhn/luhn_test.star]: https://github.com/OmegaPointZero/starlarky/blob/bc58a870acf4683607ee81934ee0ab7538a8ad8e/larky/src/test/resources/vendor_tests/luhn/luhn_test.star#L1-L3

File: larky/src/main/resources/vendor/luhn.star (hunk 1 above)
Comment (OmegaPointZero) [reply larky/src/main/resources/vendor/luhn.star]: Added with https://github.com/verygoodsecurity/starlarky/pull/125/commits/9d0f909365aa37e0d5a246914e390f501e499f11

File: larky/src/main/resources/vendor/luhn.star [hunk 4]
```diff
@@ -0,0 +1,44 @@
+load("@stdlib//larky", "larky")
+
+def _sum(nums):
+    s = 0
+    for i in nums:
+      s += i
+    return s
+   
+def _mod_digits(d):
```
[the remark this answers is not in the document]
Comment (OmegaPointZero) [reply larky/src/main/resources/vendor/luhn.star]: Replaced: https://github.com/OmegaPointZero/starlarky/blob/bc58a870acf4683607ee81934ee0ab7538a8ad8e/larky/src/main/resources/vendor/luhn.star#L10

**Comment by mahmoudimus** [comment]
@OmegaPointZero looks good to me!

**Review by mahmoudimus — APPROVED** [review]

**Pull request merged**